src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -50,6 +50,13 @@ protected internal ContractState CreateContract(byte[] script, byte[] manifest)
             if (!contract.Manifest.IsValid(hash)) throw new InvalidOperationException($"Invalid Manifest Hash: {hash}");
 
             Snapshot.Contracts.Add(hash, contract);
+
+            // Execute _deploy
+
+            ContractMethodDescriptor md = contract.Manifest.Abi.GetMethod("_deploy");
+            if (md != null)
+                CallContractInternal(contract, md, new Array(ReferenceCounter), CallFlags.All);
+
             return contract;
```

Comment:
The return value is pushed onto the current stack, which is the stack of `_deploy()`. I think we should push it onto the caller's stack.

Reply:
Done